process/smartContract/process.go

```diff
@@ -305,9 +305,8 @@ func (sc *scProcessor) doExecuteSmartContractTransaction(
 		return 0, err
 	}
 
-	var consumedFee *big.Int
 	var results []data.TransactionHandler
-	results, consumedFee, err = sc.processVMOutput(vmOutput, txHash, tx, acntSnd, vmInput.CallType, vmInput.GasProvided)
+	results, err = sc.processVMOutput(vmOutput, txHash, tx, acntSnd, vmInput.CallType, vmInput.GasProvided)
```

Comment:
All the modifications from this file should not be backwards compatible and activated with epoch flag?

Reply:
they are. as deploy of smart contract are disabled. builtIn functions are disabled. There is no SC calling another SC right now - thus vmOutput and totalConsumedFee will be OK.

Reply:
ok